### PR TITLE
Changes to run on current Swift

### PR DESCRIPTION
**Pull request by alper**
On current Swift as supplied by Xcode (i.e. Swift 4).

### CODE REVIEW

File: 2.0/docs/getting-started/manual.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ import PackageDescription
 let package = Package(
```
Comment (tanner0101) [inline 2.0/docs/getting-started/manual.md]: this should include the swift tools version header

Comment (alper) [reply 2.0/docs/getting-started/manual.md]: I don't understand. Do you have a reference?

Comment (linqingmo) [reply 2.0/docs/getting-started/manual.md]: Add ```// swift-tools-version:4.0``` at first line in Package.swift.

**Comment by 0xTim** [comment]
Given that Vapor 2 supports 3.1 and 4 should we have instructions for both?

**Pull request closed without merging**